app/lib/types.rb

```diff
@@ -53,6 +53,13 @@ module Types
     details: String
   )
 
+  Report = String.enum('caseworker_report', 'processed_report', 'workload_report')
+
+  USER_ROLE_REPORTS = {
+    CASEWORKER_ROLE => [Report['workload_report'], Report['processed_report']],
+    SUPERVISOR_ROLE => Report.values
+  }.freeze
```

Comment:
Rather than `USER_ROLE_REPORTS` are these `REPORTING_ROLES` ?

Reply:
These are the reports available to each user depending on their role. Or the user_role's reports.